customer-endpoint/src/main/kotlin/org/ostelco/prime/customer/endpoint/store/SubscriberDAOImpl.kt

```diff
@@ -190,9 +190,9 @@ class SubscriberDAOImpl : SubscriberDAO {
         }
     }
 
-    override fun provisionSimProfile(identity: Identity, regionCode: String, profileType: String?): Either<ApiError, SimProfile> {
+    override fun provisionSimProfile(identity: Identity, regionCode: String, profileType: String?, alias: String): Either<ApiError, SimProfile> {
         return try {
```

Comment:
The `return` can be replaced with `Either<ApiError, SimProfile> =`
` try { ...` as it is only one statement in the function. No bit deal though.

Reply:
I only added the `alias` parameter to the method. As @la3lma pointed out there are lot of code readability things we can do. Let's do that in a separate task after coming up with some coding guidelines.